src/sentry/static/sentry/app/views/incidents/details/header.tsx

```diff
@@ -83,74 +85,73 @@ export default class DetailsHeader extends React.Component<Props> {
+          </IncidentTitle>
+        </PageHeading>
+
         {hasIncidentDetailsError ? (
           <StyledLoadingError />
         ) : (
           <GroupedHeaderItems>
-            <HeaderItem>
-              <ItemTitle>{t('Status')}</ItemTitle>
-              <ItemValue>{this.renderStatus()}</ItemValue>
-            </HeaderItem>
-            <HeaderItem>
-              <ItemTitle>{t('Duration')}</ItemTitle>
-              {incident && (
-                <ItemValue>
-                  <Duration
-                    seconds={getDynamicText({value: duration || 0, fixed: 1200})}
-                  />
-                </ItemValue>
-              )}
-            </HeaderItem>
-            <HeaderItem>
-              <ItemTitle>{t('Users affected')}</ItemTitle>
-              {incident && (
-                <ItemValue>
-                  <Count value={incident.uniqueUsers} />
-                </ItemValue>
+            <ItemTitle>{t('Status')}</ItemTitle>
+            <ItemTitle>{t('Project')}</ItemTitle>
+            <ItemTitle>{t('Users affected')}</ItemTitle>
+            <ItemTitle>{t('Total events')}</ItemTitle>
+            <ItemTitle>{t('Duration')}</ItemTitle>
+            <ItemTitle>{t('Notifications')}</ItemTitle>
+            <ItemValue>{this.renderStatus()}</ItemValue>
+            <ItemValue>
+              {project && (
+                <Projects slugs={[project]} orgId={params.orgId}>
+                  {({projects}) => (
+                    <ProjectBadge project={projects && projects.length && projects[0]} />
+                  )}
+                </Projects>
               )}
-            </HeaderItem>
-            <HeaderItem>
-              <ItemTitle>{t('Total events')}</ItemTitle>
-              {incident && (
-                <ItemValue>
-                  <Count value={incident.totalEvents} />
-                  <OpenLink to={eventLink}>
-                    <InlineSvg src="icon-open" size="14" />
-                  </OpenLink>
-                </ItemValue>
-              )}
-            </HeaderItem>
-            <HeaderItem>
-              <ItemTitle>{t('Notifications')}</ItemTitle>
+            </ItemValue>
+            {incident && (
+              <ItemValue>
+                <Duration seconds={getDynamicText({value: duration || 0, fixed: 1200})} />
+              </ItemValue>
+            )}
+            {incident && (
               <ItemValue>
-                <SubscribeButton
-                  disabled={!isIncidentReady}
-                  isSubscribed={incident && !!incident.isSubscribed}
-                  onClick={onSubscriptionChange}
-                  size="small"
-                />
+                <Count value={incident.uniqueUsers} />
               </ItemValue>
-            </HeaderItem>
+            )}
+            {incident && (
+              <ItemValue>
+                <Count value={incident.totalEvents} />
+                <OpenLink to={eventLink}>
```

Comment:
Bowen is going to chat with Adi to see if it makes sense to add an if-else condition to link to Discover2 some time later.